libs/shared/error/src/lib/error.ts

```diff
@@ -5,8 +5,12 @@
 import { MultiError, VError, Options } from 'verror';
 
 export class BaseError extends VError {
-  constructor(options: Options | Error, message: string, ...params: any[]) {
-    super(options, message, ...params);
+  constructor(
+    options: Options | Error | string | undefined,
```

Comment:
With this change `BaseError(undefined)` is valid according to `BaseError` typings, but during runtime it rejects with an `AssertionError` (shown below). Not sure this is ideal.

```
AssertionError [ERR_ASSERTION]: first argument to VError, SError, or WError constructor must be a string, object, or Error (string) is required
```

Reply:
Fixed